### ISSUE TITLE
miniconda silent install crashes Outlook

### ISSUE THREAD

**Issue by davidanthoff** [conda/conda#1084]
When I do a silent miniconda install like this while I have Outlook 2013 x64 running

````
Miniconda3-3.7.0-Windows-x86.exe /AddToPath=0 /RegisterPython=0 /S /D=C:\somedir
````

it will crash Outlook. 100% reliably and reproducible. This is on Windows 8.1 x64.

This might be an old problem with NSIS that others have [had](http://sourceforge.net/p/nsis/bugs/1049/) [before] (http://forums.winamp.com/showthread.php?t=260624).

**Comment by stevengj** [comment]
See the comments [here](http://stackoverflow.com/questions/5317448/nsis-script-that-installs-chrome-extension-kills-outlook), which seem to suggest it was a problem with ZipDLL.

**Comment by stevengj** [comment]
It seems like you should comment on the NSIS bug if you can provide them with a simple reproducible case.

**Comment by davidanthoff** [comment]
@ilanschnell or @asmeurer, could you take a look at this issue? This is causing problems downstream with julia, where the conda package manager is now used by IJulia and a plotting package to install dependencies, but now crashes Outlook on Windows when the miniconda setup is run.

I would report this bug upstream with NSIS, but the conda repo does not seem to contain any setup logic, so I don't know what version of NSIS is used for the setup, nor do I have access to the setup script so that I could post that at the NSIS site. I think my bug report would be poor, whereas you might be able to give a proper one.

Bug I do believe that my bug report here should be good enough for you to repo the bug.

**Comment by tkelman** [comment]
Bump. Any triage here?

**Comment by ilanschnell** [comment]
Miniconda uses nsis 2.46 on Windows.

**Comment by stevengj** [comment]
(Note that the Conda package in Julia currently works around this by detecting whether Outlook is running and halting with an error message in that case, rather than crashing Outlook.  Luthaf/Conda.jl#15.)

@ilanschnell, if you can reproduce the Outlook crash, I agree with @davidanthoff that you are probably in a better position to add information to the NSIS bug because you know exactly how you set up NSIS.  We really appreciate your time and effort.

**Comment by brettcannon** [comment]
This is still a problem for Windows users. We just got a report today from someone using the miniconda 4.0.5 installer in silent mode crashing Outlook. And reporting this upstream to NSIS in hopes of a fix trickling down to miniconda is tough as [NSIS 2.46](https://sourceforge.net/projects/nsis/files/NSIS%202/2.46/) is over 6 years old and they released [NSIS 3.0 yesterday](http://nsis.sourceforge.net/News).

**Comment by kalefranz** [comment]
This issue was moved to ContinuumIO/anaconda-issues#968

**Comment by mingwandroid** [comment]
Hi all, I've fixed this bug with https://github.com/mingwandroid/nsis-untgz/commit/29e79565c3f434b6987c0067b1cc81be6bacacf8

We've verified the fix in our v4.2 test installers and the issue will not be present in the final release.
